spec/features/navigation_spec.rb

```diff
@@ -42,4 +42,16 @@
     navigation = find(".navigation")
     expect(navigation).not_to have_link("Product Meta Tags")
   end
+
+  it "shows link to resource for which index? is authorized" do
+    visit admin_customers_path
+    navigation = find(".navigation")
+    expect(navigation).to have_link("Products")
+  end
+
+  it "hides link to resource for which index? is not authorized" do
+    visit admin_customers_path
+    navigation = find(".navigation")
+    expect(navigation).not_to have_link("Orders")
+  end
```

Comment:
Nitpick: how about moving these to `spec/example_app/spec/features/authorization_spec.rb`? I think that the authorization feature, since it's optional, is differentiated enough that deserves having its own file.

Reply:
Mmm, I'd agree with this. It'll stop the file becoming too big now that we know already it's a bit different.

Reply:
Uh, now I realize that I initially created this file in the wrong path :sweat: @sedubois created it in the right place. I'll move mine in a separate PR.